docs/openstack/create-and-connect-to-the-VM/ssh-to-the-VM.md

```diff
@@ -0,0 +1,320 @@
+      $ chmod 600 authorized_keys
+    ```
+
+## How To Enable Remote Desktop Protocol Using xrdp on Ubuntu
+
+### Log in to the server with Sudo access
+
+In order to install the `xrdp`, you need to login to the server with `sudo` access
+to it.
+
+    ssh username@your_server_ip
+
+For example:
+
+    ssh ubuntu@199.94.60.66
+
+### Installing a Desktop Environment
+
+After connecting to your server using SSH update the list of available packages
+using the following command:
+
+    sudo apt update
+
+Next, install the `xfce` and `xfce-goodies` packages on your server:
+
+    sudo apt install xfce4 xfce4-goodies -y
+
+!!! info "Select Display Manager"
+    If prompted to choose a display manager, which manages graphical login mechanisms
+    and user sessions, you can select any option from the list of available display
+    managers. For instance, here we have `gdm3` as the default selection.
+
+    ![xrdp Display Manager](images/xrdp_display_manager.png)
+
+### Installing xrdp
+
+To install xrdp, run the following command in the terminal:
+
+    sudo apt install xrdp -y
+
+After installing xrdp, verify the status of xrdp using systemctl:
+
+    sudo systemctl status xrdp
+
+This command will show the status as **active (running)**:
+
+Output:
+
+    ● xrdp.service - xrdp daemon
+        Loaded: loaded (/lib/systemd/system/xrdp.service; enabled; vendor preset: enab>
+        Active: active (running) since Mon 2024-02-12 21:33:01 UTC; 9s ago
+        ...
+        CGroup: /system.slice/xrdp.service
+                └─8839 /usr/sbin/xrdp
+
+!!! info "What if xrdp is not Running?"
+    If the status of xrdp is not running, you may have to start the service manually
+    with this command: `sudo systemctl start xrdp`. After executing the above command,
+    verify the status again to ensure xrdp is in a running state.
+
+Make xrdp to use the desktop environment we previously created:
+
+    sudo sed -i.bak '/fi/a #xrdp multiple users configuration \n xfce-session \n' /etc/xrdp/startwm.sh
+
+### Configuring xrdp and Updating Security Groups
+
+If you want to customize the default xrdp configuration (optional), you will need
+to review the default configuration of xrdp, which is stored under `/etc/xrdp/xrdp.ini`.
+`xrdp.ini` is the default configuration file to set up RDP connections to the
+xrdp server. The configuration file can be modified and customized to meet the
+RDP connection requirements.
+
+Add a new security group with a RDP (port 3389) rule open to the public for an
+RDP connection and attach that security group to your instance.
+
+!!! info "How to Update Security Group(s) on a Running VM?"
+    Following [this guide](../access-and-security/security-groups.md#update-security-groups-to-a-running-vm),
+    you'll be able to attach any newly created security group(s) with all the
+    required rules to a running VM.
+
+Restart once the xrdp server to make sure all the above changes are reflected:
```

Comment:
once the xrdp server does what?
maybe:
server to -> server starts to
or
server to -> server modifications are complete to

Reply:
Restart once the xrdp server -> Restart the xrdp server